src/controllers/update/update.review.beneficial.owner.individual.ts

```diff
@@ -0,0 +1,62 @@
+import { REVIEW_BENEFICIAL_OWNER_INDEX_PARAM, UPDATE_BENEFICIAL_OWNER_BO_MO_REVIEW_URL, UPDATE_BENEFICIAL_OWNER_TYPE_URL, UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE, UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_URL } from "../../config";
+import { NextFunction, Request, Response } from "express";
+import { getApplicationData, removeFromApplicationData, setApplicationData } from "../../utils/application.data";
+import { logger } from "../../utils/logger";
+import { BeneficialOwnerIndividualKey } from "../../model/beneficial.owner.individual.model";
+import { ApplicationData, ApplicationDataType } from "../../model";
+import { setBeneficialOwnerData } from "../../utils/beneficial.owner.individual";
+import { v4 as uuidv4 } from "uuid";
+import { saveAndContinue } from "../../utils/save.and.continue";
+import { Session } from "@companieshouse/node-session-handler";
+
+export const get = (req: Request, res: Response) => {
+  logger.debugRequest(req, `${req.method} ${req.route.path}`);
+  const appData = getApplicationData(req.session);
```

Comment:
You could probably get better test coverage by checking 
if (appData?.benfitial_owners_indivdual) {
    redirect(BOMOSummaryPage)
}
and writing a test for getApplicationData mocked to retun null and return {}